README.md

```diff
@@ -73,7 +93,7 @@ We create a zip file
     $ zip -r ../../io.murano.zip *
     $ cd ./../../
 
-Finally, we execute the commando package-import for the murano client library (cosidering we are configuring murano agains FIWARE Lab)
+Finally, we execute the command package-import for the Murano client library (considering we are configuring Murano against FIWARE Lab)
```

Comment:
No no, it is not finally. Finally is the execution of the test that you make after command package import. Nevertheless, user do not see the command... (missing command).